tcms/testruns/forms.py

```diff
@@ -26,6 +26,10 @@ class BaseRunForm(forms.Form):
         required=False,
     )
 
+    class Meta:
+        model = TestRun
+        fields = ['notes', 'default_tester', 'build', 'manager', 'summary']
```

Comment:
I believe the fields listed here and the ones defined above shouldn't be duplicated, see `NewBugForm` for example.
Maybe only UserField()'s must stay b/c the widgets accept email or username. The rest shouldn't be needed.

Reply:
Also move the meta definition at the top of this class and overrides below it.